vidarrtest-regression.json.in

```diff
@@ -40,9 +40,55 @@
       {
         "metrics_calculate": "@CHECKOUT@/calculate.sh",
         "metrics_compare": "@CHECKOUT@/compare.sh",
-        "output_metrics": "/.mounts/labs/gsi/testdata/empty/output_metrics/TEST_EMPTY.metrics",
+        "output_metrics": "/.mounts/labs/gsi/testdata/empty/output_metrics/TEST_EMPTY1.metrics",
+        "type": "script"
+      }
+    ]
+  },
+  {
+    "arguments": {
+      "empty.exitCode": null,
+      "empty.n": null,
+      "empty.log.exitCode": null,
+      "empty.log.n": null,
+      "empty.log.mem": null,
+      "empty.log.timeout": null,
+      "empty.dummyInput": {
+        "contents": {
+          "configuration": "/.mounts/labs/gsi/testdata/empty/input_data/DUMMY_INPUT.txt",
+          "externalIds": [
+            {
+              "id": "TEST",
+              "provider": "TEST"
+            }
+          ]
+        },
+        "type": "EXTERNAL"
+      }
+    },
+    "description": "Log the default number of lines (10) to stderr and default exit (code=0)",
+    "engineArguments": {
+      "read_from_cache": false,
+      "write_to_cache": false
+    },
+    "id": "TEST_EMPTY2",
+    "metadata": {
+      "empty.err": {
+        "contents": [
+          {
+            "outputDirectory": "@SCRATCH@/@DATE@_Workflow_empty_TEST_EMPTY2_@JENKINSID@"
+          }
+        ],
+        "type": "ALL"
+      }
+    },
+    "validators": [
+      {
+        "metrics_calculate": "@CHECKOUT@/calculate.sh",
+        "metrics_compare": "@CHECKOUT@/compare.sh",
+        "output_metrics": "/.mounts/labs/gsi/testdata/empty/output_metrics/TEST_EMPTY2.metrics",
```

Comment:
You'll need to copy TEST_EMPTY.metrics to TEST_EMPT1.metrics and TEST_EMPTY2.metrics in /.mounts/labs/gsi/testdata/empty/output_metrics/